### PR TITLE
Add Katello 4.6 pipeline

### PR DESCRIPTION
**Pull request by sjha4**
Adds 4.6 specific pipelines and vars. Haven't removed any older configs or pipelines but let me know if we should.

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
We recently made some changes and haven't reflected that in tool_belt. Could you also add the mapping to `theforeman.org/pipelines/test/foreman/vars.groovy`? This means PRs to foreman.git will check out the proper Katello version for the stable branch.

File: theforeman.org/pipelines/test/foreman/vars.groovy [hunk 1]
```diff
@@ -21,6 +25,9 @@ def foreman_unit_versions = [
     'develop': [
         'ruby': ['2.7']
     ],
+    '3.4-stable': [
+        'ruby': ['2.7']
+    ],
     '3.3-stable': [
```
Comment (sjha4) [inline theforeman.org/pipelines/test/foreman/vars.groovy:31]: @ekohl..Thanks! Added 3.4 and 4.6 configs.

File: theforeman.org/pipelines/vars/katello/4.6.groovy [hunk 2]
```diff
@@ -0,0 +1,16 @@
+def foreman_version = '3.4'
+def katello_version = '4.6'
+def foreman_el_releases = [
+    'el7',
```
Comment (ekohl) [inline theforeman.org/pipelines/vars/katello/4.6.groovy:4]: There won't be al EL7 release
```suggestion
```

File: theforeman.org/yaml/jobs/pipeline/katello-rpm-pipeline.yaml [hunk 3]
```diff
@@ -21,4 +21,5 @@
       - '4.2'
       - '4.3'
       - '4.4'
-      - '4.5'
\ No newline at end of file
+      - '4.5'
+      - '4.6'
```
Comment (ekohl) [inline theforeman.org/yaml/jobs/pipeline/katello-rpm-pipeline.yaml:25]: Could you ensure a line ending at the end of the file?

**Review by ekohl — CHANGES_REQUESTED** [review]

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**